crawlerapp/app/pageparse.js

```diff
@@ -5,27 +5,53 @@ var request = require('request');
 
 module.exports = {
 
-    searcHelper: function(url, searchType, searchDepth, keyword) {
-        //TEST VARIABLES
-        searchType = 1;
-        searchDepth = 1;
-        keyword = "";
-
-        if (urlIsValid(url)) {
-            //TODO: CALL APPROPRIATE CRAWL BASED ON TYPE
-            getPage(url);
+    searchHelper: function(url, searchType, searchDepth, keyword) {
+        if (searchType == 0) {
```

Comment:
right now, I have search type being passed in as a string. 'bfs' and 'dfs'. This is a simple fix than can be changed in next commit.

Reply:
Change added.